src/components/Books.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import './Books.css';
+import Forms from './Forms';
+
+class Books extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <>
+        <div className="card">
+          <div className="main-area">
+            <span>Alchemist</span>
+            <h1>The Soul of the Universe</h1>
+            <p>Destiny</p>
+            <ul>
+              <li>Corn</li>
+              <li>Corn</li>
+              <li>Corn</li>
+            </ul>
+          </div>
+
+          <div className="main-area2">
```

Comment:
- [ ] KIndly separate book codes into separate component to make your code more dry and reusable. you can create a book component and import it into the books component to make your codes more professional, that book component must have a button to delete a book and takes the data as a prop and  use it to display the title and author. you got this :+1: